lava-master/Dockerfile

```diff
@@ -1,4 +1,6 @@
-FROM baylibre/lava-master-base:2019.03_stretch
+FROM lavasoftware/lava-server:2019.07
+
+RUN apt-get update && apt-get -y install lava-dispatcher sudo lava-coordinator
```

Comment:
Doesn't `apt-get update` need a `-y`?
Why is lava-dispatcher needed in the lava-master image?

Reply:
apt-get update does not need -y since it didnt ask question.
I have removed lava-dispatcher